test/middleware.ts

```diff
@@ -0,0 +1,264 @@
+var assert = require('assert')
+var http = require('http')
+var morgan = require('morgan')
+var request = require('supertest')
+var split = require('split')
+var kayvee = require('../lib/kayvee')
+var kv_middleware = require('../lib/middleware')
+
+describe('middleware', function () {
+  var expected;
+  it('should pass default fields', function (done) {
+    var cb = afterTest(2, function (err, res, line) {
+      if (err) return done(err)
+      var masked = line.replace(/response-time\":\d+/, 'response-time":99999')
+      expected = kayvee.format({
+        "method": "GET",
+        "path": "/hello/world",
+        "params": "?a=1&b=2",
+        "response-size": 12345,
+        "response-time": 99999,
+        "status-code": 200,
+        "x-forwarded-for": "foo",
+      })
+      assert.equal(masked, expected)
```

Comment:
nit: It feels a bit weird that this relies on the keys being printed in a consistent order each time instead of comparing the objects